<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr: Highlights</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Open resources for comparative genomic analysis of lentiviruses.</h2>
      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="HighlightsEiav" class="anchor" href="#HighlightsEiav" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>XRVs: Equine infectious anemia virus</strong>
	   </h3>
	   <hr>


		 <p>
 
		   <b>Equine infectious anaemia virus (EIAV)</b> is a lentivirus that infects horses worldwide.
		   In a collaborative investigation of genetic diversity among globally sampled EIAV
		   isolates we found clear evidence of geographic compartmentalization, with
		   distinct strains being predominant in Asia, Europe and the Americas.
		   In addition, our study found evidence that the majority EIAV strain
		   found in the Americas originated approximately
		   400 years ago - around the time that horses were reintroduced to
		   the New World by European colonists.
		   
		 </p>

		 <p>
 
		   Equids became extinct in the Americas ~12,500 years ago, and were not
		   re-introduced until the <b>late 15th Century</b>.
		   Prior to the development of reliable serologic tests in the early 1970s,
		   it was virtually impossible to distinguish inapparent carriers of EIAV
		   from uninfected animals and European colonists likely
		   introduced the disease to the Americas unwittingly.
		   
		 </p>


		 <p>
 
		   New World viruses analyzed in the study were sampled in countries
		   throughout the Americas, including sites in Argentina, Canada and the USA.
		   The predominance of a single, distinct EIAV strain suggests that
		   a small number of ancestral
		   '<b>founder</b>' strains gave rise to the main EIAV epidemic in the New World.	   

		 </p>

         <br>

		 <p><img src="../assets/images/eiav-shacklefordbanks-1.jpg" alt="The North-Western periphery of Europe" /></p>



<blockquote>
	<b>The wild horses of Shackleford Banks</b>: "They've always been here - they swam ashore off sinking ships."
</blockquote>


	   <br>


		 <p>
 

		   Importantly, the study included a sample obtained from a horse belonging to the
		   <a href="http://www.shacklefordhorses.org/about.htm" target="_blank"><b>Shackleford Banks</b></a>
		   wild herd.
		   The Shackleford Banks wild horses occupy an isolated barrier island
		   in North Carolina, USA. They are a unique breed, thought to be descended from
		   animals abandoned by Spanish
		   explorers in the early part of the 16th century, after failed attempts
		   to colonize the Atlantic coastal regions of North America.

		   
		 </p>


		 <p>
 
		   Phylogenetic trees place the EIAV strain found in Shackleford Banks horses
		   in an ancestral position relative to other New World isolates, consistent
		   with EIAV having been introduced by early European colonists.
		   
		 </p>


		 <h4>
		   <a id="DataItemsEiav" class="anchor" href="#DataItemsEiav" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Links to related data items:</strong>
		 </h4>


		 <p>

		   <ul>
			 <li>Annotated reference sequences for all EIAV genotypes</li>
			 <li>Genome length alignments of all EIAV genotypes</li>
			 <li>Reference-constrained alignments for all sequences</li>
		   </ul>
		   
		</p>




	   <br>



	   <h3>
		 <a id="HighlightsPSIV" class="anchor" href="#HighlightsPSIV" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERVs: Prosimian immunodeficiency viruses (PSIVs)</strong>
	   </h3>
	   <hr>




		<p>
          The human immunodeficiency viruses <b>HIV-1</b> and <b>HIV-2</b> are human
          lentiviruses that arose through cross-species transmission from primates.
          It is well established that these transmission events occurred in 
          West and Central Africa, where lentiviruses are known to circulate
          among wild apes and monkeys. However, it has generally been assumed
          that natural lentivirus infections of primates are restricted to the
          African mainland.
s
		</p>



		<p>

          In direct contradiction of this notion, we identified an endogenous 
          lentivirus in the genome of a Malagasy primate - the grey mouse lemur 
          (<i>Microcebus murinus</i> - pictured). 
          <i>Grey mouse lemur prosimian immunodeficiency virus</i> <b>(PSIVgml)</b> 
          was identified by <i>in silico</i> screening of whole genome sequence data,
          and contains some derived features, such as a <i>vif</i> gene, but lacks  
          certain genes found in simian lentiviruses, such as <i>vpr</i>.
          PSIVgml is also phylogenetically intermediate, grouping between the feline 
          and simian immunodeficiency viruses in trees.

		</p>



		 <p><img src="../assets/images/lenti-eve-psiv.jpg" alt="Lemurs" /></p>

		


<blockquote>
	Lemurs - photographs from
	<a target="blank" href="https://www.instagram.com/dukelemurcenter/">Duke Lemur Center</a>.
</blockquote>



		<p>
          The discovery of PSIVgml is striking, because all lemurs are restricted
          to Madagascar - an ancient island that has been colonized by only a small
          number of mammalian groups since it formed approximately 200 million years
          ago. This unique biogeographic history is reflected in the astonishing 
          (and sadly endangered) 
          <a href="http://www.guardian.co.uk/environment/video/2010/aug/03/lemurs-madagascar" target="_blank">Malagasy primate fauna</a>,
          consisting of numerous, morphologically diverse lemur species all of which 
          are thought to have descended from a single founder population that first 
          reached Madagascar over 60 million years ago.  

		</p>



		<p>
          In order for lentiviruses to also reach  Madagascar, they must either have
          been present in this founder population of lemurs, or have been introduced 
          via cross-species transmission from a non-primate species that colonised 
          the island at a later date. Since at least one of these scenarios has to 
          be correct, lentiviruses are either (i) more transmissible between species
          groups, or (ii) far older than has previously been recognised. 

		</p>



		<p>
          As such, it is possible that lentiviral infections afflict, or have 
          afflicted, primate populations throughout the world. Uncharacterised primate
          lentiviruses might represent a potential source of new human infections,
          but could also provide valuable new model systems for studying HIV/AIDS. 

		</p>


	   <h4>
		 <a id="DataItemsPSIV" class="anchor" href="#DataItemsPSIV" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Links to related data items:</strong>
	   </h4>



		<p>

		  <ul>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//PSIV1-con.fna">PSIV1 consensus sequence</a></li>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//PSIV2-con.fna">PSIV2 consensus sequence</a></li>			
 		  </ul>

  		</p>

	   <br>


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Lentivirus-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 
 		</p>



	   <h3>
		 <a id="HighlightsRELIK" class="anchor" href="#HighlightsRELIK" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERVs: Rabbit endogenous lentivirus K (RELIK)</strong>
	   </h3>
	   <hr>


		<p>        
          <b>Rabbit endogenous lentivirus K (RELIK)</b> is
          <a href="https://doi.org/10.1073/pnas.0807873105" target="_blank">an endogenous
          retrovirus (ERV) lineage derived from a lentivirus</a>, identified in the genome
          of the European rabbit.
          This discovery provided the first insights into the ancient history of
          the <i>Lentivirus</i> genus. <b>Molecular clock-based calibrations</b>, applied
          to pairs of segmentally duplicated RELIK insertions, demonstrate that RELIK
          entered the lagomorph germline at least 6 million years ago.
		</p>

 
		<p>            
          In spite of its ancient origin, RELIK exhibits
          many of the features found in present day lentiviruses, such as <i>tat</i> and
          <i>rev</i> genes for the regulation of viral gene expression.
		</p>


		 <p><img src="../assets/images/lenti-eve-relik.jpg" alt="A hare" /></p>


<blockquote>
	A startled-looking <b>European hare</b> (<i>Lepus europaeus</i>), photographed in Scotland by
	<a target="blank" href="https://www.instagram.com/andyparkinsonphoto/">Andy Parkinson</a>.
</blockquote>


		<p>
          A follow-up study demonstrated the presence of lentivirus insertions in
          a wide range of lagomorph species, showing that invasion of the lagomorph germline
          by lentiviruses took place prior to the divergence of rabbits and hares
          ~12 million year ago.
 		</p>

 
		<p>        
         
          The absence of RELIK insertions from the genome of the
          <a target="blank" href="https://www.nwf.org/Educational-Resources/Wildlife-Guide/Mammals/American-Pika">American pika (<i>Ochotona princeps</i>)</a>
          indicates that genome invasion occurred after
          the divergence of leporids and pikas ~25 million years ago.

		</p>

	   <h4>
		 <a id="DataItemsRELIK" class="anchor" href="#DataItemsRELIK" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Links to related data items:</strong>
	   </h4>


		<p>

		  <ul>
			<li> <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv/RELIK-con.fna">RELIK consensus sequence</a></li>
		  </ul>

  		</p>


	   <br>







	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>

		 Gifford RJ
		 <b>(2012)</b>
		 <br>
		 Viral evolution in deep time - Lentiviruses and mammals.
		 <br>
		 <i>Trends in Genetics</i>
		 <a href="https://doi.org/10.1016/j.tig.2011.11.003" target="_blank">[view]</a>


		 <br>
		 <br>
		 Capomaccio S, Cappelli K, Cook RF, Nardi F, Gifford RJ, Marenzoni ML, and F. Passamonti 
		 <b>(2011)</b>
		 <br>
		 Geographic structuring of global EIAV isolates: A single origin for New World strains? 
		 <br>
		 <i>Virus Research</i>
		 <a href="https://doi.org/10.1016/j.virusres.2011.11.011" target="_blank">[view]</a> 


		 <br>
		 <br>
		 Gilbert C, Maxfield DG, Goodman SM, and Feschotte C.
		 <b>(2009)</b>
		 Parallel germline infiltration of a lentivirus in two Malagasy lemurs.
		 <i>PLoS Genet.</i>
		 <a href="https://doi.org/10.1371/journal.pgen.1000425" target="_blank">[view]</a>


		 <br>
		 <br>
		 Gifford RJ, Katzourakis A, Tristem M, Pybus, OG, Winters M, and RW. Shafer.
		 <b>(2008)</b>
		 A transitional endogenous lentivirus from the genome of a basal primate and implications for lentivirus evolution.
		 <br>
		 <i> PNAS </i>
		 <a href="https://doi.org/10.1073/pnas.0807873105" target="_blank">[view]</a>

		 <br>
		 <br>
		 Katzourakis A, Tristem M, Pybus OG, and RJ. Gifford
		 <b>(2007)</b>
		 Discovery and analysis of the first endogenous lentivirus.
		 <i>PNAS</i>
		 <a href="https://doi.org/10.1073/pnas.0807873105" target="_blank">[view]</a>

		 <br>
		 <br>
         Reina R, Mora MI, Glaria I, García I, Solano C, Luján L, Badiola JJ, Contreras A, Berriatua E, Juste R, Mamoun RZ, Rolland M, Amorena B, and de Andrés D.
         <b>(2004)</b>
         Phylogenetic analysis and reclassification of caprine and ovine lentiviruses based on 104 new isolates: evidence for regular sheep-to-goat transmission and worldwide propagation through livestock trade.
         <i>Virology</i>
         <a href="http://www.ncbi.nlm.nih.gov/pubmed/14967484" target="_blank">[abstract]</a>


 		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



