<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Open resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>

    </section>

    <section class="main-content">

	   <h3>
		 <a id="EveProject" class="anchor" href="#EveProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The ERV component of Lentivirus-GLUE</strong>
	   </h3>
	   <hr>

	   
		<p>
          <b>Endogenous retroviruses (ERVs)</b> are retrovirus-derived DNA sequences that occur in
          the germline genomes of host species. ERVs arise when retroviruses infect germline
          cells (i.e. spermatozoa, eggs, or early embryonic cells) so that integrated retrovirus
          DNA is vertically inherited as a novel host allele.
		</p>

	   
		<p>
          ERVs provide a unique source of <b>retrospective information</b> about the
          <b>long-term history</b> and retroviruses and their hosts.
          ERVs derived from lentiviruses are very rare, and for decades they were assumed
          not to occur. However,
          as described <a href="./highlights.html"><b>here</b></a>, whole genome sequencing 
          of vertebrates in the 2000s led to the discovery of
          ERVs derived from lentiviruses in a variety of species.
          These discoveries allowed the timeline of lentivirus evolution to be reliably calibrated.
		</p>

		 
	    <br>


         <p><img src="../assets/images/lentivirus-erv-species.jpg" alt="Timeline of lentivirus evolution" /></p>



<blockquote>
         <b>Species containing lentivirus ERVs</b>: left to right - leporids (family Leporidae);
         lemurs (family Lemuridae); Sunda flying lemur (<i>Galeopterus variegatus</i>);
         mink (<i>Neovison vison</i>)
</blockquote>
		 
	    <br>
  
		<p>
          Progress in characterising ERVs has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          This greatly complicates comparative genomic analysis.
          <b>Lentivirus-GLUE</b> aims to address these issues for lentiviral ERVs.
          it applies a set of programmatic principles for systematically organising
          and refining the ERV 'fossil record',
		</p>

        <p>
          This page provides background information about the ERV component of 
          Lentivirus-GLUE, and direct links to specific data items. 

		</p>



        
	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions in Lentivirus-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Lentivirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of ERV-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/erv/lentiErvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional tables: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains information pertaining to individual
		   ERV sequences: e.g. species in which they occur, genome assembly version, 
		   genomic location(i.e. scaffold, location coordinates, and orientation).
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains information pertaining to our
		   ERV reference sequences, which we have constructed in an effort to
		   reconstruct, as closely as possible, the sequences of the progenitor viruses
		   that gave rise to ERVs. 
		</p>




	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Reference sequences and genome features</strong>
	   </h3>
	   <hr>
 
 		 <p>

		   For each lentiviral paleovirus species, we have defined a 'master' reference sequence, as follows:
		   <ul>
		   
			 <li><b>Rabbit endogenous lentivirus K</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv/RELIK-con.fna"><b>RELIK con</b></a>)</li>
			 <li><b>Prosimian immunodeficiency virus 1</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//PSIV1-con.fna"><b>PSIV1-con</b></a>)</li>
			 <li><b>Prosimian immunodeficiency virus 2</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//PSIV2-con.fna"><b>PSIV2-con</b></a>)</li>
			 <li><b>Mustelid endogenous lentivirus</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//MELV-con.fna"><b>MELV</b></a>)</li>
			 <li><b>Dermopteran endogenous lentivirus</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//DELV-con.fna"><b>DELV</b></a>)</li>
			 <li><b>Springhare endogenous lentivirus</b>: (<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/sources/erv/fasta-refseqs-erv//SpELV-con.fna"><b>SpELV</b></a>)</li>
		   </ul>

		</p>



		<p>		
          The paleovirus reference sequences represent the efforts of researchers
          to reconstruct, as closely as possible,
          the genomes of the ancestral lentiviruses that gave rise to ERV lineages. 
		  All reference sequences are linked to auxiliary data in		 
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-refseqs-side-data.tsv"><b>tabular format</b></a>.
		</p>
		

		<p>
		  We defined the known locations of lentivirus genome features on paleovirus reference sequences.
		  (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/erv/lentiErvReferences.glue">here</b></a>).
		  Putative ancestral <b>open reading frames (ORFs)</b> encoding accessory genes have been
		  identified in many endogenous lentiviruses.
		  In some cases these represent clear homologs of the accessory genes found in 
		  exogenous lentiviruses, but in others it remains unclear whether they are homologs
		  of previously described accessory genes or distinct genes.
		  We therefore extended the set of <b>lentivirus genome features</b> defined in our core project
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/erv/lentiErvFeatures.glue">to include these putatively distinct genes</a>.
		</p>
	   



	   <br>
	   <h3>
		 <a id="locusSequenceData" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus ERV sequences and sequence-associated data</strong>
	   </h3>
	   <hr>

		<p>
          The 
          <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-side-data.tsv"><strong>tabular file</strong></a>
          contains information about the genomic location of each ERV. 
          ERVs were classified by comparison to a
          <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/hepadna_refs_consolidate_final.faa">reference library of lentivirus polypeptide sequences</a>
          library designed to represent the known diversity of lentiviruses - this includes
          extinct lineages represented only by endogenous viral elements (ERVs).
 		</p>

		<p>
          ERV sequences were recovered from whole genome sequence (WGS) assemblies via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI's genomes resource</a>.
		</p>

  
		<ul>
		<li>
          Sequence-associated data in tabular format can be found <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-side-data.tsv"><strong>here</strong></a>.
 		</li>

		<li>
           Nucleotide level data in FASTA format (individual files) can be found <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/sources/fasta-digs-erv"><strong>here</strong></a>.
		</li>


		<li>
		  The complete list of <b>vertebrate</b> genomes screened can be found
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/vertebrate-genomes-screened.csv"><b>here</b></a>.
		</li>

 
		<li>
		  The set of <b>lentivirus</b> polypeptide sequences used as probes can be found
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/lentivirus-probes.digs.faa"><b>here</b></a>.
		</li>
  
		<li>
		  The final set of <b>lentivirus</b> and <b>ERV</b> polypeptide sequences used as
		  references can be found
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/lentivirus-references.digs.faa"><b>here</b></a>.
		</li>
   
		<li>
		  Input parameters for screening using the
		   <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/"><b>DIGS tool</b></a>
		  can be found
		  <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/lentivirus-screen.ctl"><b>here</b></a>.
		</li>
	
		</ul>






 

	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
	   <hr>

		<p>
          Multiple sequence alignments (MSAs) are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Lentivirus-GLUE.
 		</p>
 	    <br>


        <p><img src="../assets/images/lentivirus-msa-tree.png" alt="Alignment tree concept"/></p>
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Lentivirus-GLUE</b>.
          We used an alignment tree data structure to link alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>

 	    <br>


		<p>
          Alignments in the project include:

		  <ol>
			<li> A single ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/root/hepadnaviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major lentivirus lineages
			(including extinct lineages represented only by ERVs).</li>
			<li> ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/internal/"><strong>Genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific lentivirus genera and ERV reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/tip"><strong>Tip</strong></a>’
			alignments in which all taxa are derived from a single ERV lineage.</li>
		  </ol>


 		</p>



 	   <br>
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERV Nomenclature</strong>
	   </h3>
	   <hr>

		<p>
          We have applied a <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">systematic approach</a>
          to naming ERVs.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>


 
        <p><img src="../assets/images/nomenclature-lenti.jpg" alt="ERV Nomenclature" /></p>

 		


		<p>
          The <b>first component</b> is the classifier ‘ERV’ (endogenous retrovirus). 
		</p>

		<p>
          The <b>second component</b> is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of ERV group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
		</p>
		
		<p>
          The <b>third component</b> of the ID defines the set of host species in which the ortholog occurs.
 		</p>





	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>

		<p>
          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/erv-root"><strong>Root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/erv-lineage"><strong>ERV lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>

   
       <br>




	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>

		 Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		 <b>(2018)</b>
		 <br>
		 Nomenclature for endogenous retrovirus (ERV) loci.
		 <br>
		 <i>Retrovirology</i>
		 <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Gifford RJ
		 <b>(2012)</b>
		 <br>
		 Viral evolution in deep time - Lentiviruses and mammals.
		 <br>
		 <i>Trends in Genetics</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197521" target="_blank">[view]</a>
		 <br>
		 <br>


		 Gifford RJ, Katzourakis A, Tristem M, Pybus, OG, Winters M, and RW. Shafer.
		 <b>(2008)</b>
		 A transitional endogenous lentivirus from the genome of a basal primate and implications for lentivirus evolution.
		 <br>
		 <i> PNAS </i>
		 <a href="http://www.pnas.org/content/105/51/20362.long" target="_blank">[view]</a>

		 <br>
		 <br>
		 Katzourakis A, Tristem M, Pybus OG, and RJ. Gifford <b>(2007)</b>
		 Discovery and analysis of the first endogenous lentivirus.
		 <i>PNAS</i>
		 <a href="http://www.pnas.org/content/104/15/6261.short" target="_blank">[view]</a>

	

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



