<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Open resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">



	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of lentiviruses using Lentivirus-GLUE</strong>
	   </h3>
	   <hr>

	   <p>
		 Comparative genomic studies can directly
		 inform efforts to limit lentivirus-associated disease and develop lentivirus-based 
		 therapeutics. 
	   </p>


	   
	   <p><img src="../assets/images/lenti-header.jpg" alt="Lentivirus phylogeny" /></p>



<blockquote>

		 Left to right: HIV-1 derives from an ancestral lentivirus that
		 originated via recombination between two distinct virus strains, one that
		 infected monkeys (e.g. red-capped mangabeys, pictured)
		 and one that infected great apes (e.g.  gorillas, pictured).
		 Eventually a descendant of this recombinant virus was transmitted from 
		 Eastern chimpanzees (<i>Pan troglodytes</i>) to humans, giving rise to
		 HIV-1 (A model of the HIV-1 virion is shown). 
		 
</blockquote>

         
	   <br>

	   <p>
	     Since the emergence of the SARS-COV2 pandemic, many have become familiar with 
	     the use of virus genome data to track the spread and evolution of pathogenic viruses
	     - e.g. via tools such as <a target="_blank" href="https://nextstrain.org/ncov/gisaid/global""><b>NextStrain</b></a>. 
		 However, comparative genomic approaches can also be used to reveal the structural and functional
		 basis of virus adaptations. 
	   </p>


	   <p>
		 <b>Lentivirus-GLUE</b> provides a highly flexible, extensible platform for
		 implementing computational genomic analysis of lentiviruses
		 in an <b>efficient, standardised and reproducible</b> way.
		 It is equally suited for research purposes as it is for more operational procedures
		 (e.g. producing standardised, genome data-based reports in a public or animal health setting)
	   </p>



	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous lentiviral elements (EPVs)
		 can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>




	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus genome features</strong>
	   </h3>
	   <hr>

		<p>
		  The integrated lentiviral genome ('provirus') is flanked at either side by identical
		  long terminal repeat (<b>LTR</b>) sequences (a form referred to as a ‘provirus’), each composed of
		  functionally distinct <b>U3</b>, <b>R</b> and <b>U5</b> regions. 
	   </p>



	   <p>
		  Lentiviruses typically encode a range of ‘accessory’ genes in addition to the fundamental
		  <b><i>gag</i></b>, <b><i>pol</i></b> and <b><i>env</i></b> genes encoded by all retroviruses.
		  Among these, the <b><i>rev</i></b> gene is thought to be encoded by all lentiviruses,
		  while <i><b>tat</i></b> is encoded by all except SRLVs and FIVs. 
		</p>
		
		
		<p>
		  We defined a <b>standard set of genome features</b> for lentiviruses, reflecting current knowledge, and
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiFeatures.glue">incorporated this information into Lentivirus-GLUE</a>.
		</p>
		




	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus reference sequences</strong>
	   </h3>
	   <hr>
		
		 <p>

		   Lentivirus-GLUE contains GLUE 
		   <b>reference sequences</b>
		   for all known lentivirus species.
		   Reference sequences are linked to auxiliary data in		 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/core/lenti-reference-data.tsv"><b>tabular format</b></a>.

		 </p>
		
		 <p>

		   For each lentivirus species, we have created a 'master' reference sequence, as follows:
		   <ul>
		   
			 <li><b>Human immunodeficiency virus</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/AF033819"><b>AF033819</b></a>)</li>
			 <li><b>Equine infectious anemia virus</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M16575"><b>M16575</b></a>)</li>
			 <li><b>Small ruminant lentivirus</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001452"><b>NC_001452</b></a>)</li>
			 <li><b>Feline immunodeficiency virus </b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M25381"><b>M25381</b></a>)</li>
			 <li><b>Bovine immunodeficiency virus </b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M32690"><b>M32690</b></a>)</li>
		   </ul>


		</p>
		 
		<p>
		
		  We defined the known locations of genome features on master reference sequences
		  (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiMasterReferences.glue">here</b></a>).

		</p>
		
		


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          <a href="./alignmenttree.html">alignment tree</a>
          data structure. Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/root/lentivirus-root-gagpol.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major lentivirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific lentivirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/main-genus"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Project-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   Lentivirus-GLUE extends GLUE's 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   through the incorporation of a number of additional fields in the 
		   sequence table, and a project-specific custom table: 'isolate'.
		   These schema extensions are defined <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiSchemaExtensions.glue"><b>here</b></a>.
		   The isolate table is linked to the main 'sequence' table via the sequence ID field.
		   It contains information pertaining to viral isolates,  e.g. species sampled, date and location of sample.
		 </p>






	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>


		 Rihn, SJ, Wilson SJ, Loman NJ, Alim M, Bakker SE, Bhella D, Gifford RJ, Rixon FJ, and PD. Bieniasz. 
		 <b>(2013)</b> 
		 <br>
		 Extreme genetic fragility of the HIV-1 capsid. 
		 <br>
		 <i>PLoS Pathogens</i>
		 <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1003461" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Capomaccio S, Cappelli K, Cook RF, Nardi F, Gifford R, Marenzoni ML, and F. Passamonti 
		 <b>(2011)</b>
		 <br>
		 Geographic structuring of global EIAV isolates: A single origin for New World strains? 
		 <br>
		 <i>Virus Research</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22119901" target="_blank">[view]</a> 


		</p>
		

	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



