<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="../assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Open resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">



	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of lentiviruses using Lentivirus-GLUE</strong>
	   </h3>
	   <hr>
	   
	   
	   <p>
		 This page provides background information on the <b>virus-associated</b>
		 data items included in the project  - information about endogenous lentiviral sequence data
		 included in Lentivirus-GLUE can be found <a href="./paleoviruses.html""><b>here</b></a>.
	   </p>

	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus genome features</strong>
	   </h3>
	   <hr>

		<p>
		  In their integrated DNA form - referred to as a '<b>provirus</b>' - retrovirus genomes are
		  flanked at either side by identical <b>long terminal repeat (<b>LTR</b>)</b>
		  sequences, each of which is composed of distinct <b>U3</b>, <b>R</b> and <b>U5</b> regions. 
	   </p>


	   <br>
	   
	   <p><img src="../assets/images/lentivirus-genomes.png" alt="Lentivirus genome" /></p>

	   <br>




	   <p>
		  Lentiviruses typically encode a range of ‘accessory’ genes in addition to the fundamental
		  <b><i>gag</i></b>, <b><i>pol</i></b> and <b><i>env</i></b> genes encoded by all retroviruses.
		  Among these, the <b><i>rev</i></b> gene is thought to be encoded by all lentiviruses,
		  while <i><b>tat</i></b> is encoded by all except SRLVs and FIVs. Numerous other 
		  accessory genes have been defined, but the evolutionary relationships between these genes
		  are not well characterised. 
		</p>
		
		
		<p>
		  We defined a <b>standard set of genome features</b> for lentiviruses, reflecting current knowledge, and
		  <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiFeatures.glue">incorporated this information into Lentivirus-GLUE</a>.
		</p>
		




	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus reference sequences</strong>
	   </h3>
	   <hr>
		
		 <p>

		   Lentivirus-GLUE contains GLUE 
		   <b>reference sequences</b>
		   for all known lentivirus species.
		   Reference sequences are linked to auxiliary data in		 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/core/lenti-reference-data.tsv"><b>tabular format</b></a>.

		 </p>
		
		 <p>

		   For each lentivirus species, we have created a 'master' reference sequence, as follows:


		   <h4>
			 <a id="primateLentiReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Primate group</strong>
		   </h4>

		   <ul>
		   
			 <li>Human immunodeficiency virus type 1: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/AF033819"><b>AF033819</b></a>)</li>
		   </ul>

		   <h4>
			 <a id="equineLentiReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Equine group</strong>
		   </h4>

		   <ul>
			 <li>Equine infectious anemia virus (Am): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/AF016316"><b>AF016316</b></a>)</li>
			 <li>Equine infectious anemia virus (Eu): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/JX480632"><b>JX480632</b></a>)</li>
			 <li>Equine infectious anemia virus (As): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/AF327877"><b>AF327877</b></a>)</li>
		   </ul>

		   <h4>
			 <a id="smallRuminantLentiReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Small ruminant group</strong>
		   </h4>

		   <ul>

			 <li>Small ruminant lentivirus (genotype A): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001452"><b>NC_001452</b></a>)</li>
			 <li>Small ruminant lentivirus (genotype B): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_001463"><b>NC_001463</b></a>)</li>
			 <li>Small ruminant lentivirus (genotype C): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/AF322109"><b>AF322109</b></a>)</li>
			 <li>Small ruminant lentivirus (genotype B3): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/JF502416"><b>JF502416</b></a>)</li>
			 <li>Small ruminant lentivirus (genotype E1): (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/EU293537"><b>EU293537</b></a>)</li>
		   </ul>

		   <h4>
			 <a id="felineLentiReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Feline group</strong>
		   </h4>

		   <ul>

			 <li>Feline immunodeficiency virus: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M25381"><b>M25381</b></a>)</li>
			 <li>Feline immunodeficiency virus Pco: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/EF455613"><b>EF455613</b></a>)</li>
			 <li>Feline immunodeficiency virus Oma: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/U56928"><b>U56928</b></a>)</li>
		   </ul>


		   <h4>
			 <a id="bovineLentiReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Bovine group</strong>
		   </h4>

		   <ul>

			 <li>Bovine immunodeficiency virus: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/M32690"><b>M32690</b></a>)</li>
			 <li>Jembrana disease virus: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/U21603"><b>U21603</b></a>)</li>
		   </ul>


		</p>
		 
		<p>
		
		  We defined the locations of lentivirus genome features on master reference sequences
		  (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiMasterReferences.glue">here</b></a>).

		</p>
		
		



	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments (MSAs)</strong>
	   </h3>
	   <hr>

		<p>
          Multiple sequence alignments (MSAs) are the basic currency of comparative genomic analysis.
          MSAs constructed in this study are linked together using
          GLUE's <b>constrained MSA tree</b> data structure.
  		</p>

         
 		<p>
          A 'constrained MSA' is an alignment in which the coordinate space is defined by 
          a selected reference sequence. Where alignment members contain insertions relative
          to the reference sequence, the inserted sequences are recorded and stored
          (i.e. sequence data is never deleted).
  		</p>

         
 		<p>
           
          GLUE projects have the option of using a data structure called an <b>alignment tree</b>
          to link constrained MSAs representing different taxonomic levels,
          and we've used this approach in Lentivirus-GLUE.
 		</p>


 	    <br>

        <p><img src="../assets/images/lentivirus-msa-tree.png" alt="Alignment tree concept"/></p>
 		

<blockquote>
		<p>
		  The schematic figure above shows the '<b>alignment tree</b>' data structure 
		  currently implemented in <b>Lentivirus-GLUE</b>.
          We used an alignment tree data structure to link alignments,
          via a set of common reference sequences.
          The root alignment contains reference sequences for major clades,
          whereas all children of the
          root inherit at least one reference from their immediate parent.
          Thus, all alignments are linked to one another via our chosen set of
          master reference sequences.
		</p>
         
</blockquote>


 	    <br>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's <b>alignment tree</b> data structure. Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/root/lentivirus-root-gagpol.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major lentivirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific lentivirus genera.</li>
		  </ol>
         
 		</p>




	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus-GLUE database schema</strong>
	   </h3>
	   <hr>


		 <p>
		   Lentivirus-GLUE extends GLUE's 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   through the incorporation of a number of additional fields in the 
		   sequence table, and a project-specific custom table: 'isolate'.
		   These schema extensions are defined <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/build/core/lentiSchemaExtensions.glue"><b>here</b></a>.
		   The isolate table is linked to the main 'sequence' table via the sequence ID field.
		   It contains information pertaining to viral isolates,  e.g. species sampled, date and location of sample.
		 </p>





	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>


		 Rihn, SJ, Wilson SJ, Loman NJ, Alim M, Bakker SE, Bhella D, Gifford RJ, Rixon FJ, and PD. Bieniasz. 
		 <b>(2013)</b> 
		 <br>
		 Extreme genetic fragility of the HIV-1 capsid. 
		 <br>
		 <i>PLoS Pathogens</i>
		 <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1003461" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Capomaccio S, Cappelli K, Cook RF, Nardi F, Gifford R, Marenzoni ML, and F. Passamonti 
		 <b>(2011)</b>
		 <br>
		 Geographic structuring of global EIAV isolates: A single origin for New World strains? 
		 <br>
		 <i>Virus Research</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22119901" target="_blank">[view]</a> 


		</p>
		

	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



