<!DOCTYPE html>
<html lang="en-us">

  <head>
  
    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="website/assets/stylesheets/github-light.css" media="screen">
    
  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Open resources for comparative genomic analysis of lentiviruses.</h2>
      <a href="./website/html/viruses.html" class="btn">Viruses</a>
      <a href="./website/html/paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./website/html/highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">



	   <h3>
		 <a id="VirusProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of lentiviruses using Lentivirus-GLUE</strong>
	   </h3>
	   <hr>
	   
		 <p>
		   This is <b>Lentivirus-GLUE</b>, a GLUE
		   project supporting comparative genomic and evolutionary analysis of
		   <a target="_blank" href="https://viralzone.expasy.org/9">lentiviruses</a>
		   (family <i>Lentivirus</i>). It contains a richly annotated sequence
		   dataset for these viruses, comprised of both 
		   <a href="./website/html/viruses.html"><b>viral sequences</b></a>
		   and
		   <a href="./website/html/paleoviruses.html"><b>endogenous retroviruses (ERVs)</b></a>.
		  
		 </p>

	   
		 <p>
		   <b>Lentiviruses</b>
		   (<a target="_blank" href ="https://viralzone.expasy.org/264">genus <i>Lentivirus</i></a>)
		   are a distinctive genus of retroviruses
		   (<a target="_blank" href ="https://viralzone.expasy.org/71">family <i>Retroviridae</i></a>)
		   that cause chronic, persistent infections in mammals, including humans. 
           They are best known as the genus of retroviruses that includes 
		   <b>human immunodeficiency virus type 1 (HIV-1)</b>. 

		 </p>

  	     <br>
		 <p><img src="./website/assets/images/lentivirus-glue-header.png" alt="HIV history images"/></p>	 
<blockquote>
		 <p>

		   The pandemic HIV-1 M group
		   is thought to have originated via zoonotic transfer from chimpanzees (left).
		   HIV-1 infects human T-lymphocytes (middle) causing chronic, persistent infections .
		   The development of combination antiretroviral therapy (right) has provided options
		   for treatment of HIV infection. However, no cure exists and constant surveillance of
		   antiretroviral drug resistance is required to preserve the effectiveness of current 
		   regimens, especially in resource-limited regions.
		 </p>
</blockquote>
  	     <br>

		 <p>
		   Since the emergence of the SARS-COV2 pandemic, many have become familiar with 
		   the use of <b>virus genome data</b>  to track the spread and evolution of pathogenic viruses
		   - e.g. via tools such as <a target="_blank" href="https://nextstrain.org/ncov/gisaid/global""><b>NextStrain</b></a>. 
		   However, it is less widely appreciated that the same kinds of data sets and comparative genomic approaches
		   can also be used to explore the structural and functional basis of virus adaptations. 
		 </p>

		 <p>	   
		   The 
		   <a target="_blank" href="https://github.com/giffordlabcvr/gluetools"><b>GLUE software framework</b></a>
		   provides an extensible platform for implementing computational genomic
		   analysis of viruses in an <b>efficient, standardised and reproducible</b> way.
		   <b>GLUE projects</b> can not only incorporate all of the data items typically used in
		   comparative genomic analysis
		   (e.g. sequences, alignments, genome feature annotations) but can also represent the complex
		   semantic links between these data items via a <b>relational database</b>.
		   This 'poises' sequences and associated data for application in computational
		   analysis, minimising the requirement for labour-intensive pre-processing of datasets.
		 </p>


		 <p>
		   GLUE projects are equally suited for carrying out exploratory work 
		   (e.g. using virus genome data to investigate structural and functional properties of viruses)
		   as they are for implementing operational procedures (e.g. producing
		   <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/exampleReports/resistanceGeno1.html?hcvGlueVersion=0.1.68""><b>standardised reports</b></a>
		   in a public or animal health setting).
		 </p>      


		 <p>
			Hosting of GLUE projects in an online version control system (e.g. <b>GitHub</b>) provides
			a mechanism for their stable, collaborative development, as shown below.
		 </p>
	   
		 <p><img src="./website/assets/images/github-hosting.jpg" alt="GitHub hosting figure"/></p>




	   <h3>
	   <a id="WhatGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What is a GLUE project?</strong>
	   </h3>
       <hr>


		 <p>
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a> is an open, integrated
		   software toolkit that provides functionality for storage and interpretation of
		   sequence data. 
		 </p>

		 <p>
		   <b>GLUE</b> supports the development of <b>“projects”</b> containing the data items
		   required for comparative genomic analysis 
		   (e.g.  sequences, multiple sequence alignments, genome feature annotations,
		   and other sequence-associated data). 
		 </p>


        
         <p><img src="./website/assets/images/lentivirus-glue-project.jpg" alt="Lentivirus GLUE project figure"/></p>
         


		 <p>
		   Projects are loaded into the <b>GLUE "engine"</b>, creating a relational database
		   that represents the semantic relationships between data items.
		   This provides a robust foundation for the implementation of <b>systematic
		   comparative analyses</b> and the development of <b>sequence-based resources</b>. 
		 </p>

        
         <p><img src="./website/assets/images/lentivirus-glue-usage.jpg" alt="Using Lentivirus GLUE"/></p>
           



		 <p>		 
			The core schema of this database can be extended to accommodate
			the idiosyncrasies of different projects, and
			GLUE provides a scripting layer (based on JavaScript) for developing custom
			analysis tools. 
		 </p>

 
		   
		 <p>
		   Some examples of '<b>sequence-based resources</b>' built for viruses using GLUE include:

		 </p>
		   
		   

		 <p>

		 <ul>
		 		 
		   <li> 
			 <a target="_blank" href="http://cov-glue.cvr.gla.ac.uk/"><b>COV-GLUE</b></a>: 
			 A GLUE resource for tracking genetic variation in SARS-COV2.
			 CoV-GLUE contains a database of amino acid replacements, insertions and
			 deletions which have been observed in GISAID hCoV-19 sequences sampled from the pandemic
		   </li>
		   <br>
		   
		   <li>
			 <a target="_blank" href="http://rabv-glue.cvr.gla.ac.uk/"><b>RABV-GLUE</b></a>:
			 Tailored toward epidemiological tracking of rabies virus (RABV).
			 Includes a database of RABV sequences and metadata from NCBI, updated daily and arranged into major and minor clades, and
			 an analysis tool providing genotyping, analysis and visualisation of submitted FASTA sequences.
 		   </li>
 		   <br>
		   
		   <li>
			 <a target="_blank" href="http://hcv-glue.cvr.gla.ac.uk/"><b>HCV-GLUE</b></a>:
			 This GLUE resource aims to support analysis of drug resistance and vaccine
			 escape in hepatitis C virus (HCV).
			 A database of HCV sequences and metadata from NCBI, updated daily and arranged
			 into clades (genotypes, subtypes). As well as pre-built multiple-sequence
			 alignments of NCBI sequences, it includes an analysis tool providing genotyping,
			 drug resistance analysis and visualisation of submitted FASTA sequences.
           </li>
	   
		 </ul>

		   
		 </p>

	     <br>

	   


	   <br>
	   <h3>
	   <a id="WhyGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>What does building the Lentivirus-GLUE project offer?</strong>
	   </h3>
	   <hr>

		 
 
		 <p>
		 
			Lentivirus-GLUE offers a number of advantages for performing comparative sequence
			analysis of lentiviruses:

		   <ol>

			 <li> <strong>Reproducibility</strong>. 
			 For many reasons, bioinformatics analyses are notoriously difficult to reproduce.
			 The GLUE framework supports the implementation of fully reproducible 
			 comparative genomics through the introduction of data standards and the use 
			 of a relational database to capture the semantic links between data items.
			 </li>
			 <br>
		   
			 <li> <strong>Reusable data objects and analysis logic</strong>. 
			 For many - if not most - comparative genomic analyses, data preparation is nine
			 tenths of the battle. The GLUE framework has been designed to ensure that 
			 work spent preparing high-value data items such as multiple sequence alignments
			 need only be performed once. Hosting of GLUE projects in an online version control 
			 system such as GitHub allows for collaborative management of important data items
			 and community testing of hypotheses.
			 </li>
			 <br>


			 <li> <strong>Validation</strong>. 
			 Building GLUE projects entails mapping the semantic links between data items
			 (e.g. sequences, tabular data, multiple sequence alignments).
			 This process provides an opportunity 
			 for cross-validation, and thereby enforces a high level of data integrity.
			 </li>
			 <br>
		 
			 <li> <strong>Standardisation of the genomic co-ordinate space</strong>. GLUE
			 projects allow all sequences to utilise the coordinate space of a chosen 
			 reference sequence. Contingencies associated with insertions and deletions
			 (indels) are handled in a systematic way.
			 </li>
			 <br>
		   
			 <li> <strong>Predefined, fully annotated reference sequences</strong>:
			 This project includes fully-annotated reference sequences for major lineages
			 within the <i>Lentivirus</i> family.
			 </li>
			 <br>
		   
			 <li> <strong>Alignment trees</strong>:
			 GLUE allows linking of alignments constructed at distinct taxonomic levels
			 via an  "<b>alignment tree</b>"
			 data structure. In the alignment tree, each alignment
			 is constrained to a standard reference sequence, thus all multiple sequence
			 alignments are linked to one another via a standardised coordinate system.
			 </li>
			 
		   </ol>

		 </p>


         <br>

	   
		 <p>
		
		   Some examples of the ways in which this GLUE resource can be used include:
		   
		   <ol>
		 
			 <li> To perform comparative genomic studies across the family <i>Lentivirus</i>,
			 e.g. as part of an investigation of lentivirus diversity.
			 </li>
			 <br>
		   
		   
			 <li> To facilitate in-depth comparative investigations of any virus species or group
			 within the family <i>Lentivirus</i>.
			 </li>
			 <br>
		   
			 <li> As a source of systematically organised information about
			 <a href="./website/html/paleoviruses.html"><b>endogenous retroviruses (ERVs) derived from lentiviruses</b></a>.
			 </li>
		   
		   </ol>
		  
		 </p>



       <br>
	   <h3>
	   <a id="installGlueProject" class="anchor" href="#glueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Installing Lentivirus-GLUE</strong>
	   </h3>
       <hr>


		   
		 <p>
	
		   On computers with <a href="http://glue-tools.cvr.gla.ac.uk/#/installation">GLUE installed</a>,
		   the <b>Lentivirus-GLUE</b> project can be instantiated by navigating to the project folder,
		   initiating GLUE, and issuing the following command in the GLUE shell:
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildLentivirusCoreProject.glue</code></pre>
		 
		 </p>

        This will build the Lentivirus-GLUE core project by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/buildLentivirusCoreProject.glue"><b>this file</b></a>.
		 
	   </p>


	   <p>
		 The <i>Lentivirus</i> project can be further extended to incorporate
		 ERV sequences by executing the commands in
		 <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/buildLentivirusPaleoProject.glue"><b>this file</b></a>,
		 as follows.
         
  <pre>
  <code>Mode path: /
  GLUE> run file buildLentivirusPaleoProject.glue</code></pre>
		 		  
	   </p>



	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>

	   <p>

		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>

		 Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		 <b>(2018)</b>
		 <br>
		 Nomenclature for endogenous retrovirus (ERV) loci.
		 <br>
		 <i>Retrovirology</i>
		 <a href="https://doi.org/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Rihn, SJ, Wilson SJ, Loman NJ, Alim M, Bakker SE, Bhella D, Gifford RJ, Rixon FJ, and PD. Bieniasz. 
		 <b>(2013)</b> 
		 <br>
		 Extreme genetic fragility of the HIV-1 capsid. 
		 <br>
		 <i>PLoS Pathogens</i>
		 <a href=" https://doi.org/10.1371/journal.ppat.1003461" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Gifford RJ
		 <b>(2012)</b>
		 <br>
		 Viral evolution in deep time - Lentiviruses and mammals.
		 <br>
		 <i>Trends in Genetics</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197521" target="_blank">[view]</a>
		 <br>
		 <br>
		 
		 Capomaccio S, Cappelli K, Cook RF, Nardi F, Gifford R, Marenzoni ML, and F. Passamonti 
		 <b>(2011)</b>
		 <br>
		 Geographic structuring of global EIAV isolates: A single origin for New World strains? 
		 <br>
		 <i>Virus Research</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22119901" target="_blank">[view]</a> 
		 <br>
		 <br>


		 Gifford RJ, Katzourakis A, Tristem M, Pybus, OG, Winters M, and RW. Shafer.
		 <b>(2008)</b>
		 A transitional endogenous lentivirus from the genome of a basal primate and implications for lentivirus evolution.
		 <br>
		 <i> PNAS </i>
		 <a href="http://www.pnas.org/content/105/51/20362.long" target="_blank">[view]</a>
		 <br>
		 <br>
		 
		 Katzourakis A, Tristem M, Pybus OG, and RJ. Gifford <b>(2007)</b>
		 Discovery and analysis of the first endogenous lentivirus.
		 <i>PNAS</i>
		 <a href="http://www.pnas.org/content/104/15/6261.short" target="_blank">[view]</a>
		 <br>
		 <br>

	

	   </p>



	   <hr>
	   <h3>
		 <a id="contributors" class="anchor" href="#contributors" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Contributors</strong>
	   </h3>
	   <br>


		 <p><a href="https://giffordlabcvr.github.io/virus-evolution/index.html" target="_blank"><b>Robert J. Gifford</b></a> (<a href="mailto:robert.gifford@glasgow.ac.uk">robert.gifford@glasgow.ac.uk</a>)</p>




	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>





	 <footer class="site-footer">

	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/DIGS-tool">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>
	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>

	 </footer>

    </section>


  </body>
  
</html>
