<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr: Highlights</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Lentivirus-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 
 		</p>



	   <h3>
		 <a id="Avi.1.Neoaves" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERVs: Rabbit endogenous lentivirus K (RELIK)</strong>
	   </h3>
	   <hr>



		<p>
Rabbit endogenous lentivirus K (RELIK) - an ancient endogenous lentivirus.

		</p>



		<p>


Unlike many other retrovirus groups, lentiviruses - the genus that includes HIV-1 and HIV-2 - have not previously been identified as endogenous sequences in animal genomes. In the absence of a genomic fossil record, the lentivirus genus has been thought to have a relatively recent origin (in evolutionary terms).

		</p>



		<p>


For example, whereas some retrovirus groups appear to predate the divergence of mammalian orders more than 60 million years ago, studies of the feline and primate lentivirus subgroups have suggested they could have originated no more than 1â€“2 million years ago.

		</p>



		<p>


In 2006 we reported the first example of an endogenous lentivirus, which we call rabbit endogenous lentivirus K (RELIK), in the genome of the European rabbit. This discovery enabled us to perform the first ever investigation of lentivirus evolutionary history using genomic fossils. We used pairs of segmentally duplicated RELIK insertions to demonstrate that RELIK entered the lagomorph germ line at least 6 million years ago. In spite of its ancient origin, RELIK exhibits many of the features found in present day lentiviruses, such as tat and rev genes for the regulation of viral gene expression.


		</p>



		<p>


Invasion of the lagomorph germ line by lentiviruses took place prior to the divergence of rabbits and hares.


		</p>



		<p>


Following our discovery of RELIK - an endogenous lentivirus - in the genome of the European rabbit, we sought to investigate the distribution of endogenous lentiviruses throughout the order Lagomorpha (rabbits, hares, and pikas).

		</p>



		<p>


The general perception of lagomorphs as fecund is misleading. In fact, many species are threatened, and some - such as the riverine rabbit - are critically endangered. Consequently, broad screening of lagomorph genomes on a species-by-species basis is impractical. Nevertheless, in a collaboration with researchers at University College London, we demonstrated the presence of lentivirus insertions in a wide range of lagomorph species, including those that are too rare to sample.

		</p>



		<p>


We were able to do this by comparing the genomes of just two species - the European rabbit (Oryctolagus cuniculus) and the European hare (Lepus europaeus). A combination of molecular and bioinformatic approaches were used to demonstrate the presence of orthologous RELIK insertions (i.e. RELIK insertions at identical loci) in both hare and rabbit genomes. This established that integration occurred prior to the divergence of the Lepus and Oryctolagus genera, and thus implies the presence of RELIK insertions in a wide range of species.

		</p>



		<p>


The orthologous RELIK insertions we describe provide a new minimum age estimate (12 million years) for the lagomorph lentiviruses, and an independent demonstration of their ancient origin. However, lentivirus insertions were not detected in whole genome sequence data derived from the American pika (Ochotona princeps), suggesting that genome invasion occurred more recently than the divergence of pikas from other lagomorphs ~25 million years ago.

		</p>



		<p>

The discovery of RELIK demonstrates that endogenous lentiviruses occur in animal genomes, although they are relatively rare. Since genome-scale sequencing has only been conducted in a tiny fraction of mammalian species, it is highly likely that other endogenous (and exogenous) lentiviruses will be discovered as more genome-sequencing projects are completed.


		</p>





        <p><img src="./images/ehbv-avi.1-neoaves.jpg" alt="Background images for Avi.28-Alauda" /></p>
 		


<blockquote>
		  Left panel: the clade 'Neoaves' includes almost all modern birds, including (clockwise)
		  flamingos (order Phoenicopteriformes), parrots (order Psittaciformes),
		  songbirds (order Passeriformes) and all birds of prey.
		  Right panel: Analysis using Lentivirus-GLUE has revealed that at least four distinct
          clades (I-IV) of avilentivirus have existed during lentivirus evolution.

</blockquote>


		<p>
		  Links to related data items: 

		  <ul>
			<li> <a target="_blank" href="../sources/fasta-refseqs-ehbv/ehbv-avi.1-neoaves.fna">ERV-Avi.1-Neoaves nucleotide consensus sequences</a></li>
		    <li> <a target="_blank" href="./images/avihepadna-phylogeny.jpg">Avihepadnaviridae phylogeny with EVEs</a></li>
		    <li> <a target="_blank" href="./images/avian-phylogeny-part1.jpg">Avian phylogeny (part 1)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		    <li> <a target="_blank" href="./images/avian-phylogeny-part2.jpg">Avian phylogeny (part 2)</a> (taken from <a target="_blank" href="https://dx.doi.org/10.1038/nature15697">Prun <i>et al</i></a>)</li>
		  </ul>

  		</p>

		
		<br>
		<h3>
		  <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
		</h3>
		<hr>

		<p>
	  

	   </p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



