<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr: Highlights</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/github-light.css" media="screen">

  </head>


  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
    </section>



    <section class="main-content">


	   <h3>
		 <a id="'Highlights" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus-GLUE: Highlights</strong>
	   </h3>
	   <hr>

		<p>
          These highlights pages aim to provide a brief overview of selected data items
          contained within the <b>Lentivirus-GLUE</b> project, and to illustrate
          how it can be used to implement reproducible comparative analyses. 
 		</p>



	   <h3>
		 <a id="HighlightsRELIK" class="anchor" href="#HighlightsRELIK" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERVs: Rabbit endogenous lentivirus K (RELIK)</strong>
	   </h3>
	   <hr>


		<p>
          In 2006 we reported an endogenous retrovirus (ERV) lineage
          derived from a lentivirus, in the genome of the European rabbit.
          We named this ERV rabbit endogenous lentivirus K (RELIK). 
          This discovery of RELIK enabled us to calibrate - for the first time - the long-term evolutionary 
          timeline of lentiviruses.
          We applied molecular clock-based 
          calibrations to pairs of segmentally duplicated RELIK insertions,
          demonstrating that RELIK entered the lagomorph germline
          at least 6 million years ago. In spite of its ancient origin, RELIK exhibits
          many of the features found in present day lentiviruses, such as tat and rev genes
          for the regulation of viral gene expression.

		</p>


		 <p><img src="assets/images/lenti-eve-relik.jpg" alt="A hare" /></p>


<blockquote>
	A startled-looking European hare  (<i>Lepus europaeus</i>), photographed in Scotland by <a target="blank" href="https://www.instagram.com/andyparkinsonphoto/">Andy Parkinson</a>.
</blockquote>


		<p>
          In a follow-up study, we demonstrated the presence of lentivirus insertions in
          a wide range of lagomorph species, showing that invasion of the lagomorph germline
          by lentiviruses took place prior to the divergence of rabbits and hares
          ~12 million year ago. Also, the absence of RELIK insertions from the genome of the
          American pika (<i>Ochotona princeps</i>) indicates that genome invasion occurred after
          the divergence of leporids and pikas ~25 million years ago.

		</p>


		<p>
		  Links to related data items: 

		  <ul>
			<li> <a target="_blank" href="../sources/fasta-refseqs-erv/RELIK-con.fna">RELIK consensus sequence</a></li>
		  </ul>

  		</p>



	   <h3>
		 <a id="HighlightsPSIV" class="anchor" href="#HighlightsPSIV" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERVs: Prosimian immunodeficiency viruses (PSIVs)</strong>
	   </h3>
	   <hr>




		<p>
         The human immunodeficiency viruses HIV-1 and HIV-2 arose through
          cross-species transmission of lentiviruses from primates to humans.
          It is well established that these transmission events occurred in 
          West and Central Africa, where lentiviruses are known to circulate
          among wild apes and monkeys. However, it has generally been assumed
          that natural lentivirus infections of primates are restricted to the
          African mainland.



        <p><img src="./images/ehbv-avi.1-neoaves.jpg" alt="Background images for PSIV" /></p>
 		


<blockquote>
	PSIV

</blockquote>



		</p>



		<p>

          In direct contradiction of this notion, we identified an endogenous 
          lentivirus in the genome of a Malagasy primate - the grey mouse lemur 
          (<i>Microcebus murinus</i> - pictured). 
          <i>Grey mouse lemur prosimian immunodeficiency virus</i> <b>(PSIVgml)</b> 
          was identified by <i>in silico</i> screening of whole genome sequence data,
          and contains some derived features, such as a <i>vif</i> gene, but lacks  
          certain genes found in simian lentiviruses, such as <i>vpr</i>.
          PSIVgml is also phylogenetically intermediate, grouping between the feline 
          and simian immunodeficiency viruses in trees.

		</p>



		<p>
          The discovery of PSIVgml is striking, because all lemurs are restricted
          to Madagascar - an ancient island that has been colonized by only a small
          number of mammalian groups since it formed approximately 200 million years
          ago. This unique biogeographic history is reflected in the astonishing 
          (and sadly endangered) 
          <a href="http://www.guardian.co.uk/environment/video/2010/aug/03/lemurs-madagascar" target="_blank">Malagasy primate fauna</a>,
          consisting of numerous, morphologically diverse lemur species all of which 
          are thought to have descended from a single founder population that first 
          reached Madagascar over 60 million years ago.  

		</p>



		<p>
          In order for lentiviruses to also reach  Madagascar, they must either have
          been present in this founder population of lemurs, or have been introduced 
          via cross-species transmission from a non-primate species that colonised 
          the island at a later date. Since at least one of these scenarios has to 
          be correct, lentiviruses are either (i) more transmissible between species
          groups, or (ii) far older than has previously been recognised. 

		</p>



		<p>
          As such, it is possible that lentiviral infections afflict, or have 
          afflicted, primate populations throughout the world. Uncharacterised primate
          lentiviruses might represent a potential source of new human infections,
          but could also provide valuable new model systems for studying HIV/AIDS. 

		</p>



		<p>
		  Links to related data items: 

		  <ul>
			<li> <a target="_blank" href="../sources/fasta-refseqs-erv/PSIV1-con.fna">PSIV1 consensus sequence</a></li>
			<li> <a target="_blank" href="../sources/fasta-refseqs-erv/PSIV2-con.fna">PSIV2 consensus sequence</a></li>
			
 		  </ul>

  		</p>






	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>

		 Gifford RJ
		 <b>(2012)</b>
		 <br>
		 Viral evolution in deep time - Lentiviruses and mammals.
		 <br>
		 <i>Trends in Genetics</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197521" target="_blank">[view]</a>
		 <br>
		 <br>


		 Gifford RJ, Katzourakis A, Tristem M, Pybus, OG, Winters M, and RW. Shafer.
		 <b>(2008)</b>
		 A transitional endogenous lentivirus from the genome of a basal primate and implications for lentivirus evolution.
		 <br>
		 <i> PNAS </i>
		 <a href="http://www.pnas.org/content/105/51/20362.long" target="_blank">[view]</a>

		 <br>
		 <br>
		 Katzourakis A, Tristem M, Pybus OG, and RJ. Gifford <b>(2007)</b>
		 Discovery and analysis of the first endogenous lentivirus.
		 <i>PNAS</i>
		 <a href="http://www.pnas.org/content/104/15/6261.short" target="_blank">[view]</a>

	

 		</p>





	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



