<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="assets/stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
    
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./viruses.html" class="btn">Viruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>

    </section>

    <section class="main-content">

	   <h3>
		 <a id="Background" class="anchor" href="#Background" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Endogenous lentivirus (ERV) data</strong>
	   </h3>

		 <p>


           Until quite recently, lentiviruses were generally regarded as a ‘modern’ retroviral group.
           Approaches based on the application of a molecular clock suggested the possibility
           of relatively recent origins, with timeframes of a few hundred or thousand years
           being proposed for some subgroups. In the absence of reliable retrospective data,
           the possibility of the entire genus arising within a relatively recent timeframe
           (e.g. less than one million years) could not be dismissed. However, 
           dramatic advances in genome sequencing have accelerated the characterisation of
           <b>‘endogenous retroviruses’ (ERVs)</b> in vertebrate genomes.
           
  
		 </p>



         <p><img src="./asset/images/lentivirus-erv-species.jpg" alt="Timeline of lentivirus evolution" /></p>



<blockquote>
         <b>Species containing lentivirus ERVs</b>: left to right - leporids (family Leporidae);
         lemurs (family Lemuridae); Sunda flying lemur (<i>Galeopterus variegatus</i>);
         mink (<i>Neovison vison</i>)
</blockquote>
		 

		 <p>
	
         
           These sequences originated via ancient ‘germline incorporation’ events in
		   which lentivirus DNA sequences were integrated into chromosomal DNA of germline
		   cells and subsequently inherited as novel host alleles. 
		   
	   
           endogenous viral elements - retrovirus-derived sequences in host genomes.
           Among the diverse range of  ‘fossilized’ virus sequences that have been
           identified in vertebrate genomes are several endogenous lentiviruses.

		 </p>


		 <p>



		   Whole genome sequencing has revealed the presence of DNA sequences derived from

		 </p>


		 <p>

		   ERV sequences are in some ways
		   equivalent to lentivirus ‘fossils’ in that they provide a source of retrospective
		   information about the distant ancestors of modern retroviruses. 
		 </p>

	   <br>


	   <p>
         Reflecting this, analysis of ERVs has proven immensely informative with respect to the long-term
         evolutionary history of lentiviruses, as discussed 
         <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">here</a>.
 	   </p>


	   <h3>
		 <a id="Virology" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Relevance to molecular biological studies of lentiviruses</strong>
	   </h3>        
	   <hr>

	   <p>
          Endogenous viral sequences can inform our understanding of contemporary viruses
          in a wide variety of ways. Perhaps most importantly, EVEs allow calibration of
          the long-term evolutionary history of virus groups, which greatly influences 
          how we understand their biology.
 	   </p>

	   <p>
         Importantly, once time calibrations have been established, a far richer range of
         comparative genomic studies can be performed. By examining
         variation in the light of a known evolutionary history, these studies can provide 
         invaluable insights into the biological mechanisms through which viruses 
         replicate and spread.
	   </p>


		 
 
 	   <br>
	   <h3>
		 <a id="EveProject" class="anchor" href="#EveProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>The EVE component of Lentivirus-GLUE</strong>
	   </h3>
	   <hr>

	   
		<p>

          Currently, the distribution and diversity of lentivirus-related sequences
          in animal genomes remains incompletely characterized.
          Progress in characterising these elements has been hampered by the challenges
          encountered attempting to analyse their fragmentary and degenerated sequences. 
          <b>Lentivirus-GLUE</b> aims to address these issues.
		</p>

	   
		<p>
          We have incorporated into this project a set of principles for
          organising the lentivirus 'fossil record', and a protocol 
          through which it can be accessed and collaboratively developed.
		</p>

        <p>
          This website provides background information about the EVE component of 
          <b>Lentivirus-GLUE</b>, and direct links to specific data items. 

		</p>

 	   <br>
	   <h3>
		 <a id="Nomenclature" class="anchor" href="#Nomenclature" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Nomenclature for ERVs</strong>
	   </h3>
	   <hr>

		<p>
          We have applied a systematic approach to naming ERV, following a convention 
          developed for <a target="_blank" href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1">endogenous retroviruses</a>.
		  Each element was assigned a unique identifier (ID) constructed from a defined
		  set of components.
  		</p>


 
        <p><img src="./asset/images/nomenclature-lenti.jpg" alt="ERV Nomenclature" /></p>

 		


		<p>
          The first component is the classifier ‘ERV’ (endogenous retrovirus). 
		</p>

		<p>
          The second component is a composite of two distinct subcomponents separated by a period:
          
          (i)  the name of ERV group;
          (ii) a numeric ID that uniquely identifies the insertion.
          The numeric ID is an integer that identifies a unique insertion locus that arose as a
          consequence of an initial germline infection. Thus, orthologous copies in different
          species are given the same number. 
 		</p>
		
		<p>
          Where an EVE sequence is thought to have been duplicated within the germline
          following it's initial incorporation (e.g. via segmental duplication or transposition)
          we have appended an additional 'duplicate id' to the numeric ID, separated by a period.
          Please note that we have not yet resolved the orthologous relationships among
          sets of ERV sequences belonging to multicopy ERV lineages. We
          have therefore assigned unique duplicate IDs to each sequence within these lineages. 
		</p>
		
		<p>
          The third component of the ID defines the set of host species in which
          the ortholog occurs, or did occur prior to being deleted.
  		</p>

 
	   <br>
	   <h3>
		 <a id="Raw ERV sequences and data" class="anchor" href="#Raw_digs_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Raw ERV sequences and data</strong>
	   </h3>
	   <hr>

		<p>
          These are the raw data generated by
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">database-integrated genome screening (DIGS)</a>.
          The 
          <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-side-data.tsv"><strong>tabular file</strong></a>
          contains information about the genomic location of each EVE. 
          EVEs were classified by comparison to a
          <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/digs/hepadna_refs_consolidate_final.faa">reference library of lentivirus polypeptide sequences</a>
          library designed to represent the known diversity of lentiviruses - this includes
          extinct lineages represented only by endogenous viral elements (EVEs).
 		</p>

		<p>
          These data were obtained via
          <a target="_blank" href="https://giffordlabcvr.github.io/DIGS-tool/">DIGS</a>
          performed in vertebrate genome assemblies downloaded from 
          <a href="https://www.ncbi.nlm.nih.gov/genome/browse ">NCBI genomes</a>
          (2020-07-15).
		</p>


		<p>
           Raw data about the EVEs in tabular format can be found <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-side-data.tsv"><strong>here</strong></a>.
  		</p>

		<p>
           Nucleotide level data in FASTA format (individual files) can be found <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/sources/fasta-digs-erv"><strong>here</strong></a>.
  		</p>
		

   
         
	   <br>
	   <h3>
		 <a id="ERV reference sequences and data" class="anchor" href="#Ref_ecv_data" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>ERV reference sequences and data</strong>
	   </h3>
	   <hr>

		<p>
           We constructed consensus sequences for paleoviruses by aligning ECV
           sequences derived from the same initial germline colonisation event - i.e.
           orthologs in distinct species, and paralogs that have arisen via intragenomic
           duplication.
  		</p>

		<p>
           Reference sequence data in tabular format are <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/erv/erv-refseqs-side-data.tsv"><strong>here</strong></a>.
  		</p>
	   <br>

		<p>
           <strong>Nucleotide level data:</strong>
  		</p>
		
		<p>

		<table>
		
			<thead>
				<tr>
					<th>Paleovirus</th>
					<th>Genome (nucs)</th>
					<th>Gag AA</th>
					<th>Pol AA</th>
					<th>Env AA</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>RELIK</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/internal/RELIK-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.fna">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>pSIV</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/internal/PSIV-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/PSIV-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/PSIV-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/PSIV-pol.aln.fna">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>MELV</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/internal/MELV-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-pol.aln.fna">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>ELVgv</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/internal/ELVgv-eves.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-core.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-surface.aln.fna">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-pol.aln.fna">FASTA MSA</a></td>
				</tr>

			</tbody>
			
		</table>
		</p>
	   <br>

		<p>
           <strong>Protein level data:</strong>
  		</p>

		<p>
		
		<table>
		
			<thead>
				<tr>
					<th>Paleovirus</th>
					<th>Genome (nucs)</th>
					<th>Gag AA</th>
					<th>Pol AA</th>
					<th>Env AA</th>
				</tr>
			</thead>
		
			<tbody>

				<tr>
					<td><i>RELIK</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.faa">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>pSIV</i></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/PSIV-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/PSIV-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.faa">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>MELV</i></td>					
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/MELV-pol.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/RELIK-pol.aln.faa">FASTA MSA</a></td>
				</tr>

				<tr>
					<td><i>ELVgv</i></td>					
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-core.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-surface.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-pol.aln.faa">FASTA MSA</a></td>
					<td><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/export/ELVgv-pol.aln.faa">FASTA MSA</a></td>
				</tr>

			</tbody>
			
		</table>


 		</p>

	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>

		<p>

          Multiple sequence alignment constructed in this study are linked together using
          GLUE's <a href="./alignmenttree.html"><b>alignment tree</b></a> data structure.
          Alignments in the project include:

		  <ol>
			<li> A single ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/root/hepadnaviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major lentivirus lineages
			(including extinct lineages represented only by ERVs).</li>
			<li> ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/internal/"><strong>Genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific lentivirus genera and ERV reference sequences.
			<li> ‘<a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/tip"><strong>Tip</strong></a>’
			alignments in which all taxa are derived from a single ERV lineage.</li>
		  </ol>


 		</p>

	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>

		<p>
          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/erv-root"><strong>Root</strong></a> phylogeny (Rep)</li>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/erv-lineage"><strong>ERV lineage-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


        
	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Paleovirus-specific schema extensions in Lentivirus-GLUE</strong>
	   </h3>
	   <hr>


		 <p>
		   The paleovirus component of Lentivirus-GLUE extends GLUE's
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   to allow the capture of EVE-specific data.
		   These schema extensions are defined in 
           <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/erv/lentiErvSchemaExtensions.glue"><b>this file</b></a>
		   and comprise two additional tables: 'locus_data' and 'refcon_data'.
		   Both tables are linked to the main 'sequence' table via the 'sequenceID' field.
  		</p>


 		<p>
		   The '<b>locus_data</b>' table contains information pertaining to individual
		   EVE sequences: e.g. species in which they occur, genome assembly version, 
		   genomic location(i.e. scaffold, location coordinates, and orientation).
		</p>

 		<p>
		   The '<b>refcon_data</b>' table contains information pertaining to our
		   ERV reference sequences, which we have constructed in an effort to
		   reconstruct, as closely as possible, the sequences of the progenitor viruses
		   that gave rise to EVEs. 
		</p>




	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
		
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>

		 Gifford RJ, Blomberg B, Coffin JM, Fan H, Heidmann T, Mayer J, Stoye J, Tristem M, and WE Johnson
		 <b>(2018)</b>
		 <br>
		 Nomenclature for endogenous retrovirus (ERV) loci.
		 <br>
		 <i>Retrovirology</i>
		 <a href="https://retrovirology.biomedcentral.com/articles/10.1186/s12977-018-0442-1" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Gifford RJ
		 <b>(2012)</b>
		 <br>
		 Viral evolution in deep time - Lentiviruses and mammals.
		 <br>
		 <i>Trends in Genetics</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22197521" target="_blank">[view]</a>
		 <br>
		 <br>


		 Gifford RJ, Katzourakis A, Tristem M, Pybus, OG, Winters M, and RW. Shafer.
		 <b>(2008)</b>
		 A transitional endogenous lentivirus from the genome of a basal primate and implications for lentivirus evolution.
		 <br>
		 <i> PNAS </i>
		 <a href="http://www.pnas.org/content/105/51/20362.long" target="_blank">[view]</a>

		 <br>
		 <br>
		 Katzourakis A, Tristem M, Pybus OG, and RJ. Gifford <b>(2007)</b>
		 Discovery and analysis of the first endogenous lentivirus.
		 <i>PNAS</i>
		 <a href="http://www.pnas.org/content/104/15/6261.short" target="_blank">[view]</a>

	

 		</p>


	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



