<!DOCTYPE html>
<html lang="en-us">

  <head>
  

    <meta charset="UTF-8">
    <title>Lentivirus-GLUE by giffordlabcvr</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link rel="stylesheet" type="text/css" href="stylesheets/normalize.css" media="screen">
    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,700' rel='stylesheet' type='text/css'>
    <link rel="stylesheet" type="text/css" href="stylesheets/stylesheet.css" media="screen">
    <link rel="stylesheet" type="text/css" href="stylesheets/github-light.css" media="screen">

  </head>

  <body>
    <section class="page-header">
      <h1 class="project-name">Lentivirus-GLUE</h1>
      <h2 class="project-tagline">Resources for comparative genomic analysis of lentiviruses.</h2>

      <a href="../index.html" class="btn">Home</a>
      <a href="./paleoviruses.html" class="btn">Paleoviruses</a>
      <a href="./highlights.html" class="btn">Highlights</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/zipball/master" class="btn">Download</a>
      <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master" class="btn">View on GitHub</a>
      
    </section>

    <section class="main-content">

	   <h3>
		 <a id="GlueProject" class="anchor" href="#GlueProject" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Comparative genomic analysis of lentiviruses using Lentivirus-GLUE</strong>
	   </h3>
	   <hr>
	   
	   
	   <p>
         Lentivirus-GLUE is a data resource designed to facilitate comparative analysis of
         lentivirus genomes. It was created using the <a href="http://glue-tools.cvr.gla.ac.uk/"><b>GLUE</b></a>
         software environment. 
	   </p>
   
        <p><img src="./images/hepadna-hosts-3.png" alt="Known lentivirus host species" /></p>

<blockquote>
		<p>
		  Left to right: 
		</p>    
</blockquote>



	   <br>

	   <p>

         The Lentivirus-GLUE project not only contains data items required for
         comparative genomic analysis
         - i.e. virus genome sequences, multiple sequence alignments, genome feature annotations, and other
         sequence-associated data -
         but also, loading the project into the GLUE software framework
         creates a relational database that represents the semantic relationships
         between these data items. This creates powerful framework for performing
         <i>standardised</i> comparative analysis, via GLUE's command layer.
	   </p>


	   <p>
		   <b>NOTE</b>: those specifically interested in <b>hepatitis B virus (HBV)</b>
		   may want to investigate 
		   
		   <a target="_blank" href="https://github.com/giffordlabcvr/NCBI-HBV-GLUE"><b>HBV-GLUE</b></a>
		   and
		   <a target="_blank" href="https://github.com/giffordlabcvr/NCBI-HBV-GLUE"><b>NCBI-HBV-GLUE</b></a>
		   - this resource was developed specifically for HBV and comes
		   complete with a <b>graphical user interface (GUI)</b> that allows users to
		   browse and interrogate the underlying GLUE database. 
	   </p>




	   <br>
	   <h3>
		 <a id="virusReferenceSequences" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus reference sequences</strong>
	   </h3>
	   <hr>
		
		 <p>

		   Lentivirus-GLUE contains GLUE 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/commandModes/commandMode/root_project_reference"><b>reference sequences</b></a>
		   for all known lentivirus species.
		   Reference sequences are linked to auxiliary data in		 
		   <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/tabular/core/hepadna-ncbi-refseqs-side-data.tsv"><b>tabular format</b></a>.

		 </p>
		
		 <p>

		   For each lentivirus species, we have created a 'master' reference sequence, as follows:
		   <ul>
			 <li><b>Equine infectious anemia virus</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
			 <li><b>Small ruminant lentivirus</b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
			 <li><b>Feline immunodeficiency virus </b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
			 <li><b>Bovine immunodeficiency virus </b>: (<a target="_blank" href="https://www.ncbi.nlm.nih.gov/nuccore/NC_003977"><b>NC_003977</b></a>)</li>
		   </ul>

		 </p>
		



	   <br>
	   <h3>
		 <a id="genomeFeatures" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Lentivirus genome features</strong>
	   </h3>
	   <hr>

		<p>
          Lentiviruses have enveloped, spherical virions and a small, circular DNA
          genome ~3 kilobases (Kb) in length. The genome is characterised by a highly
          streamlined organization incorporating extensive gene overlap - the open reading
          frame (ORF) encoding the viral polymerase (P) protein occupies most of the genome
          and typically overlaps at least one of the ORFs encoding the core (C), and
          surface (S) proteins. 
 		</p>
		
		<p>
         We defined a
          <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/core/hepadnaviridaeFeatures.glue"><b>standard set of genome features</b></a>
          for lentiviruses and the locations of these genome features
		   on master reference sequences
		   (<b>see <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/core/hepadnaviridaeExplicitFeatureLocations.glue">here</b></a>).
		  
		 </p>
		


	   <br>
	   <h3>
		 <a id="Multiple sequence alignments" class="anchor" href="#MultipleSequenceAlignments" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Multiple sequence alignments</strong>
	   </h3>
	   <hr>


		<p>
          Multiple sequence alignment constructed in this study are linked together using
          GLUE's 
          <a href="./alignmenttree.html">alignment tree</a>
          data structure. Alignments in the project include:
         
		  
		  <ol>
			<li> A ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/alignments/root/hepadnaviridae.aln.fna"><strong>root</strong></a>’
			alignment constructed to represent proposed
			homologies between representative members of major lentivirus lineages</li>
			<li> ‘<a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/alignments/internal/"><strong>genus-level</strong></a>’
			alignments constructed to represent proposed homologies between the genomes of
			representative members of specific lentivirus genera.</li>
		  </ol>
         
 		</p>



	   <br>
	   <h3>
		 <a id="Phylogenetic trees" class="anchor" href="#PhylogeneticTrees" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Phylogenetic trees</strong>
	   </h3>
	   <hr>


		<p>

          We used GLUE to implement an automated process for deriving midpoint rooted,
          annotated trees from the alignments included in our project.
 		</p>


		<p>
          Trees were constructed at distinct taxonomic levels:
        
		  <ol>
			<li> Recursively populated <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/main-root"><strong>root</strong></a> phylogenies</li>
			<li> <a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/tree/master/trees/main-genus"><strong>Genus-level</strong></a> phylogenies</li>
		  </ol>

 		</p>


	   <br>
	   <h3>
		 <a id="schemaExtensions" class="anchor" href="#sequenceData" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Project-specific schema extensions</strong>
	   </h3>
	   <hr>


		 <p>
		   Lentivirus-GLUE extends GLUE's 
		   <a target="_blank" href="http://glue-tools.cvr.gla.ac.uk/#/coreSchema"><b>core schema</b></a>
		   through the incorporation of a number of additional fields in the 
		   sequence table, and a project-specific custom table: 'isolate'.
		   These schema extensions are defined <a target="_blank" href="https://github.com/giffordlabcvr/Lentivirus-GLUE/blob/master/glue/core/hepadnaviridaeSchemaExtensions.glue"><b>here</b></a>.
		   The isolate table is linked to the main 'sequence' table via the sequence ID field.
		   It contains information pertaining to viral isolates,  e.g. species sampled, date and location of sample.
		 </p>






	   <br>
	   <h3>
		 <a id="Publications" class="anchor" href="#Publications" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>Related Publications</strong>
	   </h3>
	   <hr>


		<p>
	   
		 Singer JB, Thomson EC, McLauchlan J, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 GLUE: A flexible software system for virus sequence data.
		 <br>
		 <i>BMC Bioinformatics</i>
		 <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-018-2459-9" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Zhu H, Dennis T, Hughes J, and RJ Gifford
		 <b>(2018)</b>
		 <br>
		 Database-integrated genome screening (DIGS): exploring genomes heuristically using sequence similarity search tools and a relational database.
		 <a href="https://www.biorxiv.org/content/early/2018/04/25/246835" target="_blank">[preprint]</a> 
		 <br>
		 <br>


		 Rihn, SJ, Wilson SJ, Loman NJ, Alim M, Bakker SE, Bhella D, Gifford RJ, Rixon FJ, and PD. Bieniasz. 
		 <b>(2013)</b> 
		 <br>
		 Extreme genetic fragility of the HIV-1 capsid. 
		 <br>
		 <i>PLoS Pathogens</i>
		 <a href="http://journals.plos.org/plospathogens/article?id=10.1371/journal.ppat.1003461" target="_blank">[view]</a> 
		 <br>
		 <br>

		 Capomaccio S, Cappelli K, Cook RF, Nardi F, Gifford R, Marenzoni ML, and F. Passamonti 
		 <b>(2011)</b>
		 <br>
		 Geographic structuring of global EIAV isolates: A single origin for New World strains? 
		 <br>
		 <i>Virus Research</i>
		 <a href="http://www.ncbi.nlm.nih.gov/pubmed/22119901" target="_blank">[view]</a> 


		</p>
		




	   <br>
	   <h3>
		 <a id="license" class="anchor" href="#license" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a><strong>License</strong>
	   </h3>
       <hr>

		 <p>
		   This project is licensed under the <a href="https://www.gnu.org/licenses/agpl-3.0.en.html">GNU Affero General Public License v. 3.0</a>.
		 </p>

	   <br>




	 <footer class="site-footer">
	   <span class="site-footer-owner"><a href="https://github.com/giffordlabcvr/Lentivirus-GLUE/">Lentivirus-GLUE</a> is maintained by <a href="https://github.com/giffordlabcvr">giffordlabcvr</a>.</span>

	   <span class="site-footer-credits">This page was generated by <a href="https://pages.github.com">GitHub Pages</a> using the <a href="https://github.com/jasonlong/cayman-theme">Cayman theme</a> by <a href="https://twitter.com/jasonlong">Jason Long</a>.</span>
	 </footer>

    </section>


  </body>
  
</html>



